backend/python/migrations/versions/04653332ed3a_add_story_test_changes.py

```diff
@@ -36,6 +36,10 @@ def upgrade():
         "story_translations_all", sa.Column("test_result", sa.JSON(), nullable=True)
     )
 
+    op.execute(
+        "ALTER TABLE story_translation_contents_all MODIFY COLUMN status ENUM('DEFAULT', 'ACTION_REQUIRED', 'APPROVED', 'TEST_CORRECT', 'TEST_PARTIALLY_CORRECT', 'TEST_INCORRECT') NOT NULL;"
```

Comment:
without this the status column was stuck on just `'DEFAULT', 'ACTION_REQUIRED', 'APPROVED'` after running the migration

Reply:
oops, thanks for catching + fixing this!